lib/boxr/metadata.rb

```diff
@@ -84,5 +84,26 @@ def metadata_schema(scope, template_key)
       schema
     end
 
+    def create_metadata_template(scope, templateKey, displayName, fields: [], hidden: nil)
+      uri = "#{METADATA_TEMPLATES_URI}/schema"
+      schema = {
+        templateKey: templateKey,
+        scope: scope,
+        displayName: displayName,
+        fields: fields
+      }
+      
+      schema[:hidden] = hidden unless hidden.nil?
+
+      metadata, response = post(uri, schema, content_type: "application/json")
+      metadata
+    end
+
+    def update_metadata(scope, templateKey, operations: [])
```

Comment:
What API cal lis this method for? It collides with https://github.com/cburnette/boxr/pull/24/files#diff-98848fd837e1241a5835ef605b8a6302R39 so we'll need to change the name to be more specific

Reply:
`templateKey` should be `template_key`